modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -423,7 +426,6 @@ const setup = (editor: Editor): RenderInfo => {
 
     editor.addCommand('ToggleToolbarDrawer', () => {
       OuterContainer.toggleToolbarDrawer(outerContainer);
-      // TODO: Consider firing event - TINY-6371
```

Comment:
When removing old TODOs please close the old ticket and link it to the current ticket, don't leave things dangling

Reply:
Closed the `TINY-6371` ticket as a duplicate.